service-test/src/main/java/org/lindeck/lindeckserver/LindeckServerApplication.java

```diff
@@ -0,0 +1,19 @@
+package org.lindeck.lindeckserver;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@SpringBootApplication
+@ComponentScan(basePackages = "org.lindeck")
+@EnableJpaRepositories(basePackages="org.lindeck")
+@EntityScan(basePackages = "org.lindeck")
+public class LindeckServerApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(LindeckServerApplication.class, args);
+	}
+
+}
```

Comment:
may be remove service-test and make service-api ? so after this push we can start using api

Reply:
There are could be many services in API (microservices), so test is just an example of such service.